Tasks/NodeToolV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -5,6 +5,8 @@
   "loc.instanceNameFormat": "Use Node $(versionSpec)",
   "loc.input.label.versionSpec": "Version Spec",
   "loc.input.help.versionSpec": "Version Spec of version to get.  Examples: 6.x, 4.x, 6.10.0, >=6.10.0",
+  "loc.input.label.versionFilePath": "Path of file",
```

Comment:
I would call it "Path to the .nvmrc file"

Reply:
> I would call it "Path to the .nvmrc file"

Actually, I want to keep it not depend on file extension, so any file extension can be used to provide Node version. I mean, we can use 'nodeversion.txt' file as well

Reply:
Do we really need this :) ? Anyway, I would rephrase it in a more descriptive way since you need to see help message to understand what type of file is needed